src/Conn.h

```diff
@@ -181,6 +181,8 @@ class Connection : public BroObj {
 				val_list* vl);
 	void ConnectionEvent(EventHandlerPtr f, analyzer::Analyzer* analyzer,
 				val_list vl);
+	void ConnectionEventFast(EventHandlerPtr f, analyzer::Analyzer* analyzer,
```

Comment:
Would you mind adding documentation to ConnectionEvent & ConnectionEventFast?

While just ConnectionEvent itself was kind of self-documentary, I expect that having ConnectionEvent + ConnectionEventFast will be rather confusing to people not familiar with the background :)